Dockerfile

```
@@ -1,69 +1,96 @@
+# STAGE 2: Build (Cross-Platform)
+# ============================================================================
 FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/sdk:10.0 AS build
+
 ARG TARGETARCH
 WORKDIR /src
 
+# Copy project file and restore specifically for the target architecture
 COPY ["PatchPanda.Web/PatchPanda.Web.csproj", "PatchPanda.Web/"]
 
-# Map Docker's TARGETARCH to the arch used in .NET RIDs and restore dependencies
 RUN export DOTNET_ARCH=$(case ${TARGETARCH} in \
```

Hadolint (2.14.0)

[warning] Quote this to prevent word splitting.

(SC2046)